### ISSUE TITLE
Star Rating - 'Select box' without a description

### ISSUE THREAD

**Issue by edutrul** [bugguide/bugguide#58]
Pertains to the Representation of Taxon and Photographic Quality star things.  Check for their accessibility overall.

_'Select box' without a description1.3.1 Info and RelationshipsDrop down menus (select boxes or role="listbox") should always have a description that is explicitly associated with the field to make sure that users of assistive technologies will also know what the field is for.

If the drop down menu (select box or role="listbox") has a visible description indicating the purpose of the menu, this description should be explicitly associated to the input drop down menu either as a HTML label or using the WAI-ARIA attribute 'aria-labelledby'.

If it is not possible to add a visible description, either add a mouseover text ('title' attribute) to the drop down menu or create an invisible label using the WAI-ARIA attribute 'aria-label'._
